creator-node/src/logging.js

```diff
@@ -3,13 +3,32 @@ const shortid = require('shortid')
 
 const config = require('./config')
 
+// taken from: https://github.com/trentm/node-bunyan/issues/194#issuecomment-347801909
+// since there is no official support for string-based "level" values
+// response from author: https://github.com/trentm/node-bunyan/issues/194#issuecomment-70397668
+function RawStdOutWithLevelName() {
+  return {
+    write: (log) => {
+      // duplicate log object before sending to stdout
+      const clonedLog = Object.assign({}, log)
+
+      // add new level (string) to level key
+      clonedLog.logLevel = bunyan.nameFromLevel[clonedLog.level]
+
+      const logLine = JSON.stringify(clonedLog, bunyan.safeCycles()) + '\n'
```

Comment:
once again, i see this safeCycles call in the linked examples, but no idea what it does
can you explain + add code comment?
not every linked example does this

Reply:
looking at the documentation, looks like this is to show the keyword `[Circular]` for printing out circular logs. +1 to sid, can we add a comment here describing that?

https://npmdoc.github.io/node-npmdoc-bunyan/build/apidoc.html#apidoc.element.bunyan.safeCycles